misc/futures-bounded/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "futures-bounded"
+version = "0.1.0"
+edition = "2021"
+rust-version.workspace = true
+license = "MIT"
+repository = "https://github.com/libp2p/rust-libp2p"
+keywords = ["futures", "async", "backpressure"]
+categories = ["data-structures", "asynchronous"]
+description = "Utilities for bounding futures in size and time."
```

Comment:
Not really bounding `Future`s in size, but rather the number, no?

Reply:
With size in this case I was referring to memory size, i.e. we are bounding the total member that a set of futures can consume by limiting how many we can have at a time. Has a nice ring to it I find :D

Reply:
The description above does not mention `set` or `group` or `collection`. Thus with `futures` I am confusing the `Future` added to the set.

Either way. Works for me.